git.html.markdown

```diff
@@ -79,6 +79,11 @@ this pointer will automatically update and point to the latest commit.
 HEAD is a pointer that points to the current branch. A repository only has 1 *active* HEAD.
 head is a pointer that points to any commit. A repository can have any number of heads.
 
+###Stages of Git
```

Comment:
Glad you added this here. Would it be best to switch the "committed" and "modified" and the "staged" and "committed" around to indicate files go from modified to staged to committed?